web/src/features/dataset/components/DatasetCard.tsx

```
@@ -40,8 +50,17 @@ const DatasetCard = ({
           </ul>
         </Card.Content>
         <Card.Footer className="flex justify-between">
-          <Button variant={'primary'} onClick={() => void navigate({ to: `/portal/datasets/${id}` })}>
-            {isManager ? 'Manage Dataset' : 'View Dataset'}
+          <Button
+            disabled={status !== 'Success'}
+            variant={'primary'}
+            onClick={() => void navigate({ to: viewDatasetUrl + `${id}` })}
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)